pkg/agent/proxy/types/types.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package types
+
+import (
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/vmware-tanzu/antrea/pkg/ovs/openflow"
+	k8sproxy "github.com/vmware-tanzu/antrea/third_party/proxy"
+)
+
+// ServiceInfo is the internal struct for string service information
+type ServiceInfo struct {
+	*k8sproxy.BaseServiceInfo
+	// cache for performance
+	OFTransportProtocol    openflow.Protocol
```

Comment:
There is no other protocol mean in the context, neither `port.Protocol` nor `openflow.Protocol` stresses transport, maybe just OFProtocol

Reply:
Make sense, done.